src/Codeception/Lib/Generator/Helper.php

```diff
@@ -40,6 +40,6 @@ public function produce()
 
     public function getHelperName()
     {
-        return '\\' . $this->namespace . 'Helper\\' . $this->name;
+        return (empty($this->namespace) ? '' : '\\' . $this->namespace) . '\\Helper\\' . $this->name;
```

Comment:
Other possibility could have been
`return rtrim('\\' . $this->namespace, '\\') . '\\Helper\\' . $this->name;`